_studio/hevce_hw/libmfx_h265e_plugin.map

```diff
@@ -0,0 +1,6 @@
+LIBMFXSW_PLUGIN {
```

Comment:
new file? why

Reply:
We did not have existing .map file for HEVCE plug-in

Reply:
that's a bug in linux implementation then. Could you, please, fix within this series, please? otherwise we will forget.

Reply:
We don't need one. All plugins share same map file. Please use following code snippet:

`set(MFX_LDFLAGS "${MFX_ORIG_LDFLAGS} -Wl,--version-script=${MFX_ROOTDIR}/mfx_lib/plugin/libmfxsw_plugin.map" )`

Reply:
Done